src/sqlalchemy_cratedb/compiler.py

```diff
@@ -317,3 +318,33 @@ def for_update_clause(self, select, **kw):
         warnings.warn("CrateDB does not support the 'INSERT ... FOR UPDATE' clause, "
                       "it will be omitted when generating SQL statements.")
         return ''
+
+
+class CrateIdentifierPreparer(sa.sql.compiler.IdentifierPreparer):
+    """
+    Define CrateDB's reserved words to be quoted properly.
+    """
+    # TODO: There are certainly more to add here than just `object`?
+    reserved_words = set(list(POSTGRESQL_RESERVED_WORDS) + ["object"])
```

Comment:
This list clearly needs to be expanded. Suggestions are welcome. Probably, derive the list directly from CrateDB / crash / Admin UI? Can someone come up with suggestions for retrieving / generating an exhaustive list of reserved keywords for CrateDB?

/cc @seut, @matriv, @surister

Reply:
In the past I extracted many into https://github.com/surister/cratedb-admin-alt/blob/master/src/store/crate_api/crate_lang.js but it is **incomplete**, perhaps we could maintain a crate-owned Gist with all of this properly done and complete. It could be javascript or python, dicts and lists are generally very copy-paste friendly.

Reply:
Yeah, it would be sweet to maintain such a list on behalf of a Python generator, which derives them directly from the ANTLR grammar or so.

Reply:
Do you think [cratedb-sqlparse](https://github.com/crate/cratedb-sqlparse) should be used here, or would it be too heavy? Can you think of any alternatives?

/cc @matriv, @seut